<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>overlay_editor</class>
 <widget class="QWidget" name="overlay_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="powder_tab">
      <attribute name="title">
       <string>Powder</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="powder_overlay_editor_layout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="laue_tab">
      <attribute name="title">
       <string>Laue</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="laue_overlay_editor_layout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mono_rotation_series_tab">
      <attribute name="title">
       <string>Mono Rotation Series</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="mono_rotation_series_overlay_editor_layout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="blank_tab">
      <attribute name="title">
       <string>Blank</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
